writers/mongodb/messages_test.go

```diff
@@ -26,7 +26,7 @@ import (
 var (
 	port       string
 	addr       string
-	testLog    = log.New(os.Stdout)
+	testLog    = log.New(os.Stdout, "info")
```

Comment:
Strign consts should be defined at the top of the file.

Reply:
Good catch here, I have made this change to all files similar to this as well!
Ex: ws/api/transport_test.go